specification/keyvault/Security.KeyVault.BackupRestore/routes.tsp

```diff
@@ -0,0 +1,256 @@
+using TypeSpec.Versioning;
+using Azure.Core;
+using Azure.ClientGenerator.Core;
+
+namespace KeyVault;
+
+/**
+ * Returns the status of full backup operation
+ */
+#suppress "@azure-tools/typespec-azure-core/use-standard-operations" "Foundations.Operation is necessary for Key Vault"
+#suppress "@azure-tools/typespec-azure-core/use-standard-names" "Operation name is already established"
+@route("/backup/{jobId}/pending")
+@get
+op fullBackupStatus is Azure.Core.Foundations.Operation<
+  {
+    /**
+     * The id returned as part of the backup request
+     */
+    @path
+    jobId: string;
+  },
+  FullBackupOperation,
+  {},
+  KeyVaultError
+>;
+
+/**
+ * Creates a full backup using a user-provided SAS token to an Azure blob storage container.
+ */
+#suppress "@azure-tools/typespec-azure-core/use-standard-operations" "Foundations.Operation is necessary for Key Vault"
+@pollingOperation(fullBackupStatus)
+@finalOperation(fullBackupStatus)
+@useFinalStateVia("azure-async-operation")
+@route("/backup")
+@post
+op fullBackup is Azure.Core.Foundations.Operation<
+  {
+    /**
+     * Azure blob shared access signature token pointing to a valid Azure blob container where full backup needs to be stored. This token needs to be valid for at least next 24 hours from the time of making this call.
+     */
+    @body azureStorageBlobContainerUri?: SASTokenParameter;
+  },
+  {
+    @statusCode _: 202;
+
+    /**
+     * The URI to poll for completion status.
+     */
+    @pollingLocation
+    @header("Azure-AsyncOperation")
+    azureAsyncOperation?: string;
+
+    /**
+     * The recommended number of seconds to wait before calling the URI specified in Azure-AsyncOperation.
+     */
+    @header("Retry-After") retryAfter?: int32;
+
+    /**
+     * The backup operation.
+     */
+    @body response: FullBackupOperation;
+  },
+  {},
+  KeyVaultError
+>;
+
+/**
+ * Pre-backup operation for checking whether the customer can perform a full backup operation.
+ */
+#suppress "@azure-tools/typespec-azure-core/use-standard-operations" "Foundations.Operation is necessary for Key Vault"
+@added(KeyVault.Versions.`v7.6_preview.1`)
+@pollingOperation(fullBackupStatus)
+@finalOperation(fullBackupStatus)
+@useFinalStateVia("azure-async-operation")
+@route("/prebackup")
+@post
+op preFullBackup is Azure.Core.Foundations.Operation<
+  {
+    /** Optional parameters to validate prior to performing a full backup operation. */
+    @body preBackupOperationParameters?: PreBackupOperationParameters;
+  },
+  {
+    @statusCode _: 202;
+
+    /**
+     * The URI to poll for completion status.
+     */
+    @pollingLocation
+    @header("Azure-AsyncOperation")
+    azureAsyncOperation?: string;
+
+    /**
+     * The recommended number of seconds to wait before calling the URI specified in Azure-AsyncOperation.
+     */
+    @header("Retry-After") retryAfter?: int32;
+
+    /**
+     * The backup operation.
+     */
+    @body response: FullBackupOperation;
```

Comment:
Same question actual response body for all 202 responses - is it actually returned by the service?

Reply:
Using recordings, I can confirm these are actually returned